### PR TITLE
updated eng docs to use same layout classes as design docs

### PR DESCRIPTION
**Pull request by MelSumner**
## :pushpin: Summary

If merged, this PR formats the engineering docs with the same layout classes as the design docs page.

## :camera_flash: Screenshots

image

### :speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

Examples: 
- issue (ux,non-blocking): These buttons should be red, but let's handle this in a follow-up.
- suggestion (non-blocking): Let's change this wording to make it easier to understand.
- issue (blocking): We shouldn't introduce this kind of tech debt; let's pair and resolve the issue in a more sustainable way.

### CODE REVIEW

File: ember-flight-icons/tests/dummy/app/templates/engineering.hbs
Comment (amyrlam) [inline ember-flight-icons/tests/dummy/app/templates/engineering.hbs:21]:
```suggestion
<p>The component comes with the following defaults:
```

File: ember-flight-icons/tests/dummy/app/templates/engineering.hbs
Comment (amyrlam) [inline ember-flight-icons/tests/dummy/app/templates/engineering.hbs:34]:
```suggestion
<p>This makes the base, required invocation quite terse- <kbd class="ds-kbd">@name</kbd> is the only property that requires specification.
```

suggestion (non-blocking): there are a few of these, we could add linting down the road? maybe a bad copy pasta if want to fix here

suggestion (non-blocking): would be cool to get rid of the "`" displaying in `<code>` and `<pre>`

happy to help with either task if you want a hand!

**Review by amyrlam — APPROVED** [review]
:shipit: lgtm, if you're waiting for brian's review, all good too!

File: ember-flight-icons/tests/dummy/app/templates/engineering.hbs
Comment (MelSumner) [reply ember-flight-icons/tests/dummy/app/templates/engineering.hbs:34]: > suggestion (non-blocking): would be cool to get rid of the "`" displaying in code and pre tags 

@amyrlam could you explain what you mean by this one?

Comment (MelSumner) [reply ember-flight-icons/tests/dummy/app/templates/engineering.hbs:34]: also, I'll run a formatter on the primary branch and update everything at once 👍 Thanks for catching these unsightly details.

**Comment by Dhaulagiri** [comment]
No need to block on my account. It'd be great to get these landed and merged into main and we can iterate on additional polish there.

File: ember-flight-icons/tests/dummy/app/templates/engineering.hbs
Comment (amyrlam) [reply ember-flight-icons/tests/dummy/app/templates/engineering.hbs:34]: right now the back tick literally displays:

image

image

//

as for formatting, sounds good, would be great to have it done in CI separately (not blocking this branch) down the road

**Pull request merged**
